app-backup/mylvmbackup/mylvmbackup-0.16.ebuild

```diff
@@ -26,6 +29,9 @@ PATCHES=(
 src_install() {
 	default
 
-	keepdir /var/tmp/${PN}/{backup,mnt}
-	fperms 0700 /var/tmp/${PN}/
+	dotmpfiles "${FILESDIR}/${PN}.conf"
```

Comment:
needs a revbump (affects installed files) but you also need to tag https://bugs.gentoo.org/840960